mava/components/jax/training/model_updating.py

```diff
@@ -130,8 +129,7 @@ def model_update_minibatch(
                     gradients[agent_key]
                 )
                 agent_metrics[agent_key]["norm_updates"] = optax.global_norm(updates)
-                metrics[agent_key] = agent_metrics
-            return (params, opt_states), metrics
+            return (params, opt_states), agent_metrics
```

Comment:
totally makes sense, have we had a look at what this does to the graphs?

Reply:
Nope not yet, but we agree that we need to remove metric dictionary since it is unnecessary! If you can check with the team in the standup tomorrow the updates of the graph if this will be merged🙏

Reply:
I'll do some quick runs to see how/if this affects any graphs.

Reply:
It definitely fixes what it is supposed to and cleans up the per agent trainer metric logging on neptune.